pkg/interface/src/views/components/Author.tsx

```diff
@@ -8,29 +8,29 @@ import OverlaySigil from "./OverlaySigil";
 import { Group, Association } from "~/types";
 import GlobalApi from "~/logic/api/global";
 import { useHistory } from "react-router-dom";
+import useLocalState from "~/logic/state/local";
 
 interface AuthorProps {
   contacts: Contacts;
   ship: string;
   date: number;
   showImage?: boolean;
-  hideAvatars: boolean;
-  hideNicknames: boolean;
   children?: ReactNode;
   unread?: boolean;
   group: Group;
   api: GlobalApi;
 }
 
 export default function Author(props: AuthorProps) {
-  const { contacts, ship = '', date, showImage, hideAvatars, hideNicknames, group, api } = props;
+  const { contacts, ship = '', date, showImage, group, api } = props;
+  const hideNicknames = useLocalState(state => state.hideNicknames);
   const history = useHistory();
   let contact;
   if (contacts) {
     contact = ship in contacts ? contacts[ship] : null;
   }
   const color = contact?.color ? `#${uxToHex(contact?.color)}` : "#000000";
-  const showNickname = !props.hideNicknames && contact?.nickname;
+  const showNickname = !hideNicknames && contact?.nickname;
```

Comment:
This pattern should be refactored out into its own hook

Reply:
Like...`useNickname(contact)`?

Reply:
Yep